VRCFaceTracking/MainStandalone.cs

```diff
@@ -95,23 +87,9 @@ public static void Initialize()
             if (!bindResults.senderSuccess)
                 Logger.Error("Socket failed to bind to sender port, please ensure it's not already in use by another program or specify a different one instead.");
 
-            var _relevantParams_v1 = UnifiedTracking.AllParameters_v1.SelectMany(p => p.GetBase()).Where(param => param.Relevant);
-            var _relevantParams_v2 = UnifiedTracking.AllParameters_v2.SelectMany(p => p.GetBase()).Where(param => param.Relevant);
-            _relevantParams = _relevantParams_v1.Concat(_relevantParams_v2);
-
-            ConfigParser.OnConfigLoaded += () =>
+            ConfigParser.OnConfigLoaded += (count) =>
             {
-                _relevantParams_v1 = UnifiedTracking.AllParameters_v1.SelectMany(p => p.GetBase())
-                    .Where(param => param.Relevant);
-                _relevantParams_v2 = UnifiedTracking.AllParameters_v2.SelectMany(p => p.GetBase())
-                    .Where(param => param.Relevant);
-
-                _relevantParams = _relevantParams_v1.Concat(_relevantParams_v2);
-
-                Logger.Msg("Config file parsed successfully! " + _relevantParams.Count() + " parameters loaded.");
-                
-                if (_relevantParams_v1.Count() > 0)
-                    Logger.Warning(_relevantParams_v1.Count() + " legacy parameters loaded. These are undocumented and outdated parameters.");
```

Comment:
Noticed that the loaded event doesn't have a way to distinguish if a v1 parameter was loaded, and the legacy message would always load even when the newer v2 parameters get loaded. Could have the lambda return something to indicate that the relevant v1 params got loaded (could be a lowest version loaded int or flags to indicate parameter versions loaded), perhaps they could be all contained in a ConfigInfo struct or something and sent as the lambda parameter?

Unless you are thinking of not having the legacy message at all?